include/date/date.h

```diff
@@ -404,8 +406,8 @@ class year
     CONSTCD11 explicit operator int() const NOEXCEPT;
     CONSTCD11 bool ok() const NOEXCEPT;
 
-    static CONSTCD11 year min() NOEXCEPT;
-    static CONSTCD11 year max() NOEXCEPT;
+    static CONSTCD11 year min() NOEXCEPT { return year{-32767}; }
+    static CONSTCD11 year max() NOEXCEPT { return year{32767}; }
```

Comment:
Inlining method definitions to work around https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53473